From 066b889695316c2bf45c95ad80f793d72ec0db75 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Fri, 8 Jul 2011 15:01:26 +1000
Subject: Remove option DeviceName.

If you can set Option DeviceName, you should also be able to set Identifier
instead.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>

diff --git a/src/xf86MuTouch.c b/src/xf86MuTouch.c
index 290fa11..a3fec4b 100644
--- src/xf86MuTouch.c
+++ src/xf86MuTouch.c
@@ -1135,11 +1135,6 @@ xf86MuTInit(InputDriverPtr	drv,
   /* Process the common options. */
   xf86ProcessCommonOptions(pInfo, pInfo->options);
 
-  str = xf86FindOptionValue(pInfo->options, "DeviceName");
-  if (str) {
-    pInfo->name = strdup(str);
-  }
-  xf86Msg(X_CONFIG, "MuTouch X device name: %s\n", pInfo->name);
   priv->screen_no = xf86SetIntOption(pInfo->options, "ScreenNo", 0);
   xf86Msg(X_CONFIG, "MuTouch associated screen: %d\n", priv->screen_no);
   priv->max_x = xf86SetIntOption(pInfo->options, "MaxX", 3000);
-- 
cgit v0.10.2

